Automattic/wp-calypso
# Plans 2023: biannual toggle foundation, behind feature flag

Fixes https://github.com/Automattic/wp-calypso/issues/74260

Part of addressing pdgrnI-28f-p2

## Proposed Changes

- Introduces the 1-year/2-year toggle behind a feature flag `plans/biannual-toggle`
    - The price description for biannual plans reads _"per month, [PRICE] billed every two years"_
- No discounted prices shown in main price displays (addressed in separate PR)

### Media

<img width="650" alt="Screenshot 2023-03-10 at 2 03 49 PM" src="https://user-images.githubusercontent.com/1705499/224311442-9b901b8f-3340-4171-a23a-2b458102944a.png">

<img width="650" alt="Screenshot 2023-03-10 at 2 21 22 PM" src="https://user-images.githubusercontent.com/1705499/224314765-dd7cf5e0-7504-46cc-bddd-10a414425b57.png">


On `/plans/2yearly/[SITE_URL]` "Plans" should be selected in navbar:

<img width="650" alt="Screenshot 2023-03-10 at 2 13 01 PM" src="https://user-images.githubusercontent.com/1705499/224313529-3e887c10-6d85-4a23-8ceb-755448ffac7b.png">

### Notes

2-year plans are also selectable/visible if visiting `/plans/2yearly/[SITE_URL]` without feature flag (same with `trunk`).

## Testing Instructions



- go to `/plans/[SITE_URL]` with the feature flag `&flags=plans/biannual-toggle`
- confirm 1-year/2-year toggle shows, as in the media
- ensure prices are correct through to the checkout page
- **(NEW)** visit `/plans/2yearly/[SITE]` and ensure sums are correct in the Price descriptions (should be `price * 24`)
- **(NEW)** visit `/plans/2yearly/[SITE]` with a site on a paid plan (e.g. Business) and ensure the `Your plan` badge is visible, and the button sends to `Manage plan`
- ensure on visiting `/plans/2yearly/[SITE_URL]` "Plans" is selected in navbar, as in media (with/out feature flag)
- disable the feature flag and ensure the monthly toggle shows instead

## Pre-merge Checklist
